### ISSUE TITLE
dataset issue

### ISSUE THREAD

**Issue by qdd1234** [jiahaoLjh/HumanDepth#5]
Hello
Thanks for your great works,I have some questions about dataset as followed:
1.How can I get the R,T which are parameters of camera from your annotations 
2.The 3d coordinate is word coordinate or camera coordinate？
If you can answer my questios，I will be grateful！

**Comment by jiahaoLjh** [comment]
Hi @qdd1234 

The 3d coordinates in the annotation are already in the camera coordinate space so that R and T are not required.

**Comment by qdd1234** [comment]
Thanks! In your annotation if I use 2d coordinate  to predict 3d coordinate , The final coordinate  is camera coordinate ？

**Comment by jiahaoLjh** [comment]
Yes. We only predict the depth of each person, and the depth is defined with respect to the camera space.

**Comment by qdd1234** [comment]
I have another question：If I want to predict 3d coordinate  from 2d coordinate ，I need to normalize 3d coordinate . In your code, I know how to normalize 2d coordinate，but how can I normalize the coordinates of the third dimension instead of using your depth formulation？Thanks!

**Comment by jiahaoLjh** [comment]
@qdd1234 What do you mean by normalizing 3d coordinate?

**Comment by qdd1234** [comment]
Normalize 3D coordinates to the interval [0,1] directly including 2d coordinate and the coordinates of the third dimension

**Comment by jiahaoLjh** [comment]
@qdd1234 Estimating 3D poses is beyond the scope of this work. If you are interested in the root-relative 3D pose, using a relative depth range w.r.t. the root joint depth, e.g. -1m to +1m, would be a common choice.

**Comment by qdd1234** [comment]
Hi I want to ask what is the function of file subject_1_cpn.h5？ joints_2d_cpn is 2d coordinate predicted by CPN？

**Comment by jiahaoLjh** [comment]
@qdd1234 

Yes
